threedi_api_qgis_client/widgets/wizard.py

```diff
@@ -0,0 +1,323 @@
+import os
+import sys
+import json
+from math import ceil
+from datetime import datetime
+from qgis.PyQt.QtSvg import QGraphicsSvgItem
+from qgis.PyQt import uic
+from qgis.PyQt.QtGui import QColor
+
+from qgis.PyQt.QtWidgets import QWizardPage, QWizard, QGridLayout, QGraphicsScene, QSizePolicy, QInputDialog
+try:
+    import pyqtgraph as pg
+except ImportError:
+    main_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    deps_path = os.path.join(main_dir, "deps", "pyqtgraph-0.10.0-py3-none-any.whl")
+    sys.path.append(deps_path)
+    import pyqtgraph as pg
+
+try:
+    from dateutil.relativedelta import relativedelta
+except ImportError:
+    main_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    deps_path = os.path.join(main_dir, "deps", "python_dateutil-2.8.1-py2.py3-none-any.whl")
+    sys.path.append(deps_path)
+    from dateutil.relativedelta import relativedelta
+
+
+base_dir = os.path.dirname(os.path.dirname(__file__))
+uicls_p1, basecls_p1 = uic.loadUiType(os.path.join(base_dir, 'ui', 'page1.ui'))
+uicls_p2a, basecls_p2a = uic.loadUiType(os.path.join(base_dir, 'ui', 'page2a.ui'))
+uicls_p2b, basecls_p2b = uic.loadUiType(os.path.join(base_dir, 'ui', 'page2b.ui'))
+uicls_p3, basecls_p3 = uic.loadUiType(os.path.join(base_dir, 'ui', 'page3.ui'))
+uicls_p4, basecls_p4 = uic.loadUiType(os.path.join(base_dir, 'ui', 'page4.ui'))
+
+
+def set_widget_background_color(widget, hex_color='#F0F0F0'):
+    palette = widget.palette()
+    palette.setColor(widget.backgroundRole(), QColor(hex_color))
+    widget.setPalette(palette)
+
+
+class Page1Widget(uicls_p1, basecls_p1):
```

Comment:
Can you maybe rename Page[X] to something more explanatory like PageFinalSummaryWidget, PagePrecipitationWidget...

Reply:
Done.